frontend/src/sdk/InitializeSDK.tsx

```diff
@@ -13,7 +14,7 @@ const InitializeSDK: React.FC = ({ children }) => {
   // TODO(SDK): Don't override children
   // TODO(SDK): AppInitSDK doesn't work with no redux
   // react_devtools_backend.js:4061 Error: could not find react-redux context value; please ensure the component is wrapped in a <Provider>
-  return <AppInitSDK configurations={appConfigurations}>{children as any}</AppInitSDK>;
+  return <AppInitSDK configurations={{ ...appConfigurations, apiDiscovery: initAPIDiscovery }}>{children as any}</AppInitSDK>;
```

Comment:
@karelhala NIT - but why aren't we adding `apiDiscovery` to `useAppConfiguration` itself versus spreading here? It would simplify things and stay consistent with what we have today.

Reply:
We'll want to adjust the apiDiscovery with some static models etc, this will ease things up.